src/module/ModbusBridge.cpp

```diff
@@ -284,4 +301,68 @@ void ModbusBridge::stop()
         m_onDeviceStatusChange(device.second, DeviceStatus::Status::OFFLINE);
     }
 }
+
+void ModbusBridge::platformStatus(ConnectivityStatus status)
+{
+    LOG(TRACE) << METHOD_INFO;
+
+    // Look if the status has changed
+    if (m_connectivityStatus == status)
```

Comment:
I think we should rather use a flag whether this is the first time we handle this instead of having a none value in enum